client/landing/stepper/declarative-flow/internals/steps-repository/domains/style.scss

```diff
@@ -243,7 +243,8 @@
 
 // Domain upsell flow
 
-.domain-upsell.domains {
+.domain-upsell.domains,
+.onboarding-pm-dev.domains {
```

Comment:
```suggestion
.onboarding-pm.domains {
```

sounds like

Reply:
sorry, the other way. sounds like [this](https://github.com/Automattic/wp-calypso/pull/77801/files#diff-5bc8c2fe5f3c46f9140fb3c093ad37430873fc9825b0bad5f7d1c263bd5b3021R140) should be updated instead